webapi/src/main/scala/org/knora/webapi/store/triplestore/embedded/JenaTDBActor.scala

```diff
@@ -132,9 +132,10 @@ class JenaTDBActor extends Actor with ActorLogging {
       * Submits a SPARQL query to the embedded Jena TDB store and returns the response as a [[SparqlSelectResponse]].
       *
       * @param queryString the SPARQL request to be submitted.
+      * @param useInference not supported.
```

Comment:
Does the parameter `useInference` ever take any effect then? If I understand correctly, inference is always activated for graphdb (unless "GRAPH <http://www.ontotext.com/explicit>" is present) but not for fuseki?

I understood that inference is handled in the SPARQL queries themselves, depending on the triplestore used. So I do not understand the usefulness of this parameter sent to the store manager.